src/EncompassRest/EncompassRestClient.cs

```diff
@@ -84,7 +84,8 @@ public static EncompassRestClient CreateFromAccessToken(string apiClientId, stri
         private readonly SemaphoreSlim _semaphoreSlim = new SemaphoreSlim(1);
 
         private HttpClient _httpClient;
-        private Loans.Loans _loans;
+		private TimeSpan _TimeOut = new TimeSpan(0, 0, 100);
```

Comment:
For private fields use camelCasing preceded with an underscore. Also, I'd recommend using `TimeSpan.FromSeconds(100)` instead.